pkgs/development/web/cypress/default.nix

```
@@ -1,4 +1,20 @@
-{ stdenv, lib, fetchzip, autoPatchelfHook, xorg, gtk2, gnome2, gtk3, nss, alsa-lib, udev, unzip, wrapGAppsHook, mesa }:
+{
+  alsa-lib,
+  autoPatchelfHook,
+  callPackage,
+  fetchzip,
+  gnome2,
+  gtk2,
+  gtk3,
+  lib,
+  mesa,
+  nss,
+  stdenv,
+  udev,
+  unzip,
+  wrapGAppsHook,
+  xorg,
+}:
```

Those inputs are formated wrong.

---

This works really well. It produces the smallest subsequent diffs and is easy to sort with your editor.
I've seen all sorts of styles in Nixpkgs. Do we have a set of guidelines nowadays?

---

```suggestion
{ alsa-lib
, autoPatchelfHook
, callPackage
, fetchzip
, gnome2
, gtk2
, gtk3
, lib
, mesa
, nss
, stdenv
, udev
, unzip
, wrapGAppsHook
, xorg
}:
```
this produces the smallest possible diff.